tests/testthat/output/trunc_mat/long_unk-5-30.txt

```diff
@@ -5,4 +5,4 @@
 3     3
 4     4
 5     5
-# ... with more rows
+with at least 5 rows total
```

Comment:
I bet this is an NA somewhere - that's the print behaviour for database tables where we don't know max